relay-metrics/src/lib.rs

```diff
@@ -68,13 +68,15 @@
     html_favicon_url = "https://raw.githubusercontent.com/getsentry/relay/master/artwork/relay-icon.png"
 )]
 
-mod aggregation;
+mod aggregator;
+mod aggregatorservice;
 mod bucket;
 mod protocol;
 mod router;
 mod statsd;
 
-pub use aggregation::*;
+pub use aggregator::*;
```

Comment:
```suggestion
```

Reply:
hmm, this would require some refactor of the benchmarking, either moving the benchmarks to the aggregator module, or benchmark the process from you send a message to merge to the service

Reply:
Can you add a comment that this is only public for benchmarks?

Reply:
yeah, did that now. 

Maybe we should do benchmarks of private functions within the file itself like unit tests in the future?